h/config.py

```diff
@@ -60,6 +62,10 @@
     EnvSetting('nsq.reader.addresses', 'NSQ_READER_ADDRS'),
     EnvSetting('nsq.writer.address', 'NSQ_WRITER_ADDR'),
     EnvSetting('origins', 'ALLOWED_ORIGINS'),
+    EnvSetting('rabbitmq.address', 'RABBITMQ_ADDRESS'),
+    EnvSetting('rabbitmq.user', 'RABBITMQ_USER'),
+    EnvSetting('rabbitmq.password', 'RABBITMQ_PASSWORD'),
+    EnvSetting('rabbitmq.vhost', 'RABBITMQ_VHOST'),
```

Comment:
I'm not a big fan of this part, but I changed it to make sure that linking a docker container works with.

Otherwise we could just reuse the `BROKER_URL` environment variable for the `h.realtime` module.

Any thoughts? Can we decide not to care about supporting linking docker containers and it magically working? (Similar to what we already do with postgres).